### PR TITLE
Displaying warining/video overwrite

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**

### CODE REVIEW

File: TranscodingDemo/Server/static/js/index.js [hunk 1]
```diff
@@ -100,8 +100,15 @@ function onUploadCompleted(xhr, data) {
     else
     {
         reponseText = JSON.parse(xhr.responseText);
-        $("#upload-board").replaceWith(uploadTemplate);
-        setWarning(reponseText)
+        if(reponseText.error.force_reset_upload)
+        {
+            window.location.replace("/OnlineTranscoding/?reset_upload=True");
```
Comment (weaselix) [inline TranscodingDemo/Server/static/js/index.js]: It is not good idea to redirect user if a file is not a valid video. Just parametrize a template and, I think, that we should decide that if video is invalid then nothing was uploaded.

Comment (nieznanysprawiciel) [reply TranscodingDemo/Server/static/js/index.js]: This is other case, when user have 2 cards open and video is valid. Uploader must reject all requests that overwrite previous file, unless flag force_reupload is set. But I couldn't find better way to force FineUploader to send this flag, then redirect to page, that generates javascript with this flag.

File: TranscodingDemo/Server/uploader/views.py [hunk 2]
```diff
@@ -47,10 +47,22 @@ def post(self, request, *args, **kwargs):
...
                     state_manager = session.get_state_manager(request)
+
+                    # Reuploading file will overwrite previous file. Check if user is aware of that.
+                    if state_manager.state == StateFlag.UploadingCompleted:
+                        is_forced_reupload = request.POST.get("force_reupload") == 'True'
+                        if not is_forced_reupload:
+                            logger.info("Trying to start new upload without 'force_reupload' set. [job_id = {}]".format(job.id))
+                            
+                            return FailResponse(status_code=HTTPStatus.BAD_REQUEST,
+                                error_msg={ 'message': 'Uploading will overwrite previous video.', 'force_reset_upload': True},
```
[the remark this answers is not in the document]
Comment (nieznanysprawiciel) [reply TranscodingDemo/Server/uploader/views.py]: I added **kwargs to FailResponse

File: TranscodingDemo/Server/static/js/index.js (hunk 1 above)
Comment (nieznanysprawiciel) [reply TranscodingDemo/Server/static/js/index.js]: I changed force_reset_upload to more meaningfull name redirect_to_reset_upload

**Pull request merged**